### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ community.
 ### Participation Conduct
 
 In order to ensure everyone has a fair, pleasant, and inclusive experience contributing to Neutrino, we ask that you
-abide by our [community participation guidelines](/contributing/code-of-conduct.md), based on the
+abide by our [community participation guidelines](/docs/contributing/code-of-conduct.md), based on the
```
Comment (eliperelman) [inline CONTRIBUTING.md]: Links for gitbook have to be relative in order for them to work on the docs site as well as github. Change to: `./code-of-conduct.md`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ possible.
 ## Getting started
 
 Most contributions will involve working with the neutrino-dev codebase. Please refer to the [development
-documentation](/contributing/development.md) for technical details on getting started.
+documentation](/docs/contributing/development.md) for technical details on getting started.
```
Comment (eliperelman) [inline CONTRIBUTING.md]: Links for gitbook have to be relative in order for them to work on the docs site as well as github. Change to: `./development.md`.

File: CONTRIBUTING.md (hunk 1 above)
Comment (helfi92) [inline CONTRIBUTING.md]: @eliperelman `/docs/contributing/README.md` is the file that the gitbook reads and in there the paths are relative. Is `CONTRIBUTING.md` also read by the gitbook? I don't see that link in `/docs/SUMMARY.md`.

Comment (eliperelman) [inline CONTRIBUTING.md]: CONTRIBUTING.md in the repo root maps to `docs/contributing/README.md` in the docs.

Comment (eliperelman) [inline CONTRIBUTING.md]: So to answer your question better, CONTRIBUTING.md is not read my gitbook, it uses contributing/README.md which is the same thing.

Comment (helfi92) [inline CONTRIBUTING.md]: I see, sounds good. Changed it back to relative paths

**Comment by helfi92** [comment]
The links don't seem to have been fixed. Does it work for you @eliperelman?

**Comment by eliperelman** [comment]
The docs haven't been deployed since master points to the v5 docs, and we can't deploy until we publish v5.

**Comment by helfi92** [comment]
Sounds good.

**Pull request merged**